.github/workflows/backend.yml

```diff
@@ -35,11 +35,11 @@ jobs:
         dfx deploy CanCan
     - name: Test
       run: |
-        for f in service/tests/*.test.sh; do
+        (for f in service/tests/*.test.sh; do
           echo "==== Run test $f ===="
           dfx canister call CanCan reset 'variant { script = 0 }'
-          ic-repl -r http://localhost:9000 "$f" || break
```

Comment:
This doesn't fail because the exit status of the for loop is the last command in the for loop, which is the (successful) `break`.

Reply:
Thanks for this fix.

In bash, I get used to doing the idiom `set -x` to get this meaning without having to write the code with `|| break` suffixes.  I wonder if that works in a CI script -- I haven't tried.